geoportailv3/static/js/draw/featurepopupservice.js

```diff
@@ -159,6 +163,8 @@ app.FeaturePopup.prototype.show = function(feature, opt_anchor) {
   goog.array.forEach(headers, function(element) {
     this.setDraggable(angular.element(element));
   }, this);
+  // Enable the dropdown menu
+  $('.dropdown-toggle')['dropdown']();
```

Comment:
@pgiraud I would like to have your advice about that point.
As the popup element is dynamically inserted, the dropdown menu inside the popup is not automatically initialized.
That means the dropdown menu doesn't open.
I found this way to bypass the issue.

Is there a better alternative ?